DigitalLearningSolutions.Web/Views/CompetencyAssessments/SupervisorSignoffDeclaration.cshtml

```diff
@@ -0,0 +1,97 @@
+    ViewData["Title"] = "Supervisor sign off declaration";
+}
+<div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+        @if (errorHasOccurred)
+        {
+            <vc:error-summary order-of-property-names="@(new[] { nameof(Model.CustomText) })" />
+        }
+<form method="post">
+    <fieldset class="nhsuk-fieldset">
+        <legend class="nhsuk-fieldset__legend nhsuk-fieldset__legend--l">
+            <h1 class="nhsuk-fieldset__heading">
+                Supervisor sign off declaration
+            </h1>
+        </legend>
+
+        <p class="nhsuk-hint">
+            This will be displayed to the supervisor when they are signing off a completed self assessment.
+        </p>
+
+        <div class="nhsuk-radios nhsuk-radios--conditional">
+
+            <div class="nhsuk-radios__item">
+                <input class="nhsuk-radios__input" id="declaration-default" name="DeclarationValue" type="radio" value="0"
+                       aria-controls="conditional-declaration-default"
+                       aria-expanded="true"
+                       @(Model.DeclarationValue == 0 ? "checked" : "")>
+                <label class="nhsuk-label nhsuk-radios__label" for="declaration-default">
+                    Default
+                </label>
+               
+            </div>
+            <div class="nhsuk-radios__conditional nhsuk-radios__conditional--hidden" id="conditional-declaration-default">
+                <div class="nhsuk-form-group">
+                    <label class="nhsuk-label" for="default-declaration-text">
+                        Declaration text
+                    </label>
+                            <div class="nhsuk-body" id="default-declaration-text">
+                                @Html.Raw(Model.DefaultText)
+                            </div>
+                </div>
+            </div>
+
+            <div class="nhsuk-radios__item">
+                <input class="nhsuk-radios__input" id="declaration-custom" name="DeclarationValue" type="radio" value="1"
+                aria-controls="conditional-declaration-custom"
+                aria-expanded="true"
+                       @(Model.DeclarationValue == 1 ? "checked" : "")>
+                <label class="nhsuk-label nhsuk-radios__label" for="declaration-custom">
+                    Custom
+                </label>
+            </div>
+
+            <div class="nhsuk-radios__conditional nhsuk-radios__conditional--hidden" id="conditional-declaration-custom">
+                <div class="nhsuk-form-group">
+                    <label class="nhsuk-label" for="custom-declaration-text">
+                        Declaration text
+                    </label>
+                            <nhs-form-group nhs-validation-for="CustomText">
+
+                    <textarea class="nhsuk-textarea" id="custom-declaration-text" name="CustomText" rows="5" aria-describedby="custom-declaration-hint">@Model.CustomText</textarea>
```

Comment:
We should use Jodit to turn this into a html input and then sanitise before storing.